pkg/vault/vault.go

```diff
@@ -196,3 +196,23 @@ func RemoveClient(clientId string, cfg *config.Config) error {
 
 	return nil
 }
+
+func GetSecrets(cfg *config.Config) (map[string]string, error) {
+	if cfg.NoCache {
+		v, err := newVault(cfg, false, nil)
+		if err != nil {
+			return nil, err
+		}
+		return v.Secrets, nil
+	}
+
+	var secrets map[string]string
+	err := callServer(cfg, true, false, func(c *rpc.Client) error {
+		return c.Call("RemoteVault.GetSecrets", struct{}{}, &secrets)
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return secrets, nil
```

Comment:
I guess you could make it just a `return callServer(...)`, just like @wojtii suggested here:
https://github.com/nordcloud/mfacli/pull/5/files#r405040197